strr-web/components/bcros/form-section/property/Address.vue

```diff
@@ -22,59 +22,118 @@
         </UFormGroup>
       </div>
       <div class="flex flex-row justify-between w-full mb-[40px] mobile:mb-[16px]">
-        <UFormGroup name="Address" class="d:pr-[16px] flex-grow">
+        <UFormGroup
+          name="streetNumber"
+          class="d:pr-[16px] flex-grow"
+          :error="streetNumberError"
+        >
+          <UInput
+            :id="streetNumberId"
+            v-model="streetNumber"
+            :aria-label="t('createAccount.contactForm.streetNumber')"
+            :placeholder="t('createAccount.contactForm.streetNumber')"
+            @input="
+              onAddressInput(false); emit('resetFieldError', [
+                'streetNumber', 'streetName', 'city', 'province', 'postalCode'
+              ])
+            "
+            @click="addressComplete(false)"
+            @blur="emit('validateStreetNumber')"
+            @change="emit('validateStreetNumber')"
+          />
+        </UFormGroup>
+        <UFormGroup
+          name="streetName"
+          class="d:pr-[16px] flex-grow"
+          :error="streetNameError"
+        >
+          <UInput
+            :id="streetNameId"
+            v-model="streetName"
+            :aria-label="t('createAccount.contactForm.streetName')"
+            :placeholder="t('createAccount.contactForm.streetName')"
+            @input="
+              onAddressInput(true); emit('resetFieldError', [
```

Comment:
Do we need to call `onAddressInput()` ? It might be tricky with my validation refactoring.

Reply:
When a new address is being inputted and autocomplete triggers then previously set fields like `city`, `province` and other need to be reset. This is essentially what's happening with `onAddressInput`.

Reply:
Hmm, shouldn't it be cleared only after I select an address from auto-complete? What if I change my mind and don't select anything, then those fields are gone?

Reply:
Fields are not being cleared now.